gui/scripts/syncthing/app.js

```diff
@@ -11,7 +11,7 @@
 var syncthing = angular.module('syncthing', [
     'angularUtils.directives.dirPagination',
     'pascalprecht.translate',
-
+    'JSONedit',
```

Comment:
+1
Getting the following error https://docs.angularjs.org/error/$injector/modulerr?p0=syncthing&p1=Error:%20%5B$injector:modulerr

(like the idea tho)